ibis/backends/sql/compilers/base.py

```diff
@@ -393,45 +379,50 @@ class SQLGlotCompiler(abc.ABC):
         ops.Uppercase: "upper",
     }
 
-    BINARY_INFIX_OPS = (
-        # Binary operations
-        ops.Add,
-        ops.Subtract,
-        ops.Multiply,
-        ops.Divide,
-        ops.Modulus,
-        ops.Power,
+    BINARY_INFIX_OPS = {
+        # Numeric
+        ops.Add: (sge.Add, True),
```

Comment:
The bool here seems like it should be a property of the operation instead of a flag.

It's hard to tell what the boolean is referring to unless you know where it's being used.

What do you think about sticking a class attribute on the ops for this property?